app/assets/stylesheets/common/foundation/base.scss

```diff
@@ -7,6 +7,7 @@
   --topic-body-width-padding: #{$topic-body-width-padding};
   --topic-avatar-width: #{$topic-avatar-width};
   --d-border-radius: initial;
+  --d-border-radius-large: initial;
```

Comment:
So this is essentially saying:

"No border radius, unless specified in a theme/theme-component?

Reply:
Yup, that's the idea. Good?